docs/docs/community/release-process.md

```diff
@@ -0,0 +1,68 @@
+---
+id: release-process-branching-strategy
+title: Release process and branching strategy
+description: Release process and branching strategy
+---
+
+# The Nebari release process
+
+This page describes the high-level details of cutting a new Nebari release.
+
+|        Cadence        |               Versioning System                |                                                                                    Release Checklist                                                                                     |                                                                                        Testing Checklist                                                                                         |
+| :-------------------: | :--------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| 3rd week of the month | [CalVer](#calver-details) - (i.e. `2022.10.1`) | [Link to issue template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=release-checklist.md&title=%5BRELEASE%5D+%3Cversion%3E) | [Link to issue template](https://github.com/nebari-dev/nebari/issues/new?assignees=&labels=type%3A+release+%F0%9F%8F%B7&template=testing-checklist.md&title=Testing+checklist+for+%3Cversion%3E) |
+
+## Release Captain responsibilities
+
+For every release, there is an assigned "Release Captain". The Release Captain's responsibilities are:
+
+- Manage both the release and testing processes and update the checklists as needed.
+- Communicate the status of the release to the rest of the Nebari development team and the community (through updating the checklists and adding status updates as comments).
+- Assign owners to checklist items (if not owned by the Release Captain).
+- Adjust the schedule, particularly the publishing dates, based on defects found, fixes made, holidays, vacations, and so on.
+- Ensure a new branch and milestone is created after completing a release cycle.
```

Comment:
I believe this statement is no longer applicable since we are using `develop` now - so there is no need to create a new branch only the milestone

Reply:
This is intermediate release branch is part of the gitlab gitflow and is what I proposed in the [original notion doc](https://www.notion.so/quansightlabs/GitFlow-Workflow-8596ceff436740e8adad9fddb328848b). 

There are two situations where this release branch comes in handy. In both instances, the `develop` branch has commits that shouldn't be included in the next release. First and most clear example is for hotfixes. Second is when there are commits made to `develop` that are not to be included in this upcoming release but rather the next release. 

`develop` branch commit history
```
commit 3 - for the dec release 
commit 2 - for the jan release 
commit 1 - for the dec release
```

Without an intermediate branch (say `release/2022.12.1`), all three of these commits are going to make into the December release which may not be what we want. With this intermediate branch, we can cherry-pick `commit 1` and `commit 3` from `develop` and cut the release with only the commits we need. 

There is an alternative and that would be to hold off merging PRs until the current release has been cut. This alternative unfortunately relies on everyone knowing which PRs belong to which release (and these things may change).

Reply:
Ignore this, I misunderstood the line this was referring to. We're in agreement :sweat_smile: